pkg/preflights/output.go

```diff
@@ -46,6 +49,23 @@ func (o Output) PrintTable() {
 	logrus.Infof("%s\n", tb.Render())
 }
 
+func (o Output) SaveToDisk() error {
+	// Store results on disk of the host that ran the preflights
+	data, err := json.MarshalIndent(o, "", "  ")
+	if err != nil {
+		return fmt.Errorf("failed to marshal preflight results: %w", err)
```

Comment:
We have been trying to keep these error messages standardised with "unable to", I might be the only one still doing so as a lot of things are happening so take this with a grain of salt.

Reply:
Yer. A number of these are `copilot` generated error messages.